From 00593d3878f6f83f48cb6bb3923f3a3fc5c5c3ff Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Fri, 19 Oct 2018 13:22:12 +0300
Subject: [PATCH 685/767] staging: fsl-dpaa2/mac: Add more PHY modes

Add support for CAUI, USXGMII and 1000BASE-X.

Signed-off-by: Florin Chiculita <florinlaurentiu.chiculita@nxp.com>
Signed-off-by: Valentin Catalin Neacsu <valentin-catalin.neacsu@nxp.com>
Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.12.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/mac/mac.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/staging/fsl-dpaa2/mac/mac.c b/drivers/staging/fsl-dpaa2/mac/mac.c
index c55a06e3a50c..a8120c96abc6 100644
--- a/drivers/staging/fsl-dpaa2/mac/mac.c
+++ b/drivers/staging/fsl-dpaa2/mac/mac.c
@@ -76,6 +76,9 @@ static phy_interface_t dpaa2_mac_iface_mode[] =  {
 	PHY_INTERFACE_MODE_QSGMII,	/* DPMAC_ETH_IF_QSGMII */
 	PHY_INTERFACE_MODE_XGMII,	/* DPMAC_ETH_IF_XAUI */
 	PHY_INTERFACE_MODE_XGMII,	/* DPMAC_ETH_IF_XFI */
+	PHY_INTERFACE_MODE_XGMII,        /* DPMAC_ETH_IF_CAUI */
+	PHY_INTERFACE_MODE_XGMII,       /* DPMAC_ETH_IF_1000BASEX */
+	PHY_INTERFACE_MODE_XGMII,       /* DPMAC_ETH_IF_USXGMII */
 };
 
 static void dpaa2_mac_link_changed(struct net_device *netdev)
-- 
2.17.0

